app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginLeft="36dp"
+    android:layout_marginRight="36dp"
+    android:orientation="vertical"
+    android:theme="@style/AppTheme">
+
+    <!--App logo-->
+    <ImageView
+        android:layout_width="150dp"
+        android:layout_height="150dp"
+        android:layout_gravity="center"
+        android:layout_marginTop="100dp"
+        android:src="@drawable/logo"/>
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="30dp"
+        android:text="Having trouble in remembering your medicines? Set alarms and also save your medical reports online through DR. Help"
+
+        android:textColor="#212121"
+        android:textSize="16sp"/>
+
```

Comment:
Please add a small comment here. Please check the same throughout in this whole PR.